interfaces/udev/backend.go

```diff
@@ -59,7 +60,7 @@ func (b *Backend) Setup(snapInfo *snap.Info, devMode bool, repo *interfaces.Repo
 	if err != nil {
 		return fmt.Errorf("cannot obtain expected udev rules for snap %q: %s", snapName, err)
 	}
-	glob := fmt.Sprintf("70-%s.rules", interfaces.SecurityTagGlob(snapName))
+	glob := fmt.Sprintf("70-%s.rules", snap.SecurityTag(snapName))
```

Comment:
Something here is bogus. The content map has data both for hooks and for a global "snap-wide" rules file.

Do we want to merge the rules for the individual hooks in the same file as well? If so, that needs doing and consideration below. If not, this is broken because the glob now misses entries which may be written by ensureDirState.